Add ipmx_pass_through_delay constraint (#29)
AMWA-TV/nmos-parameter-registers: capabilities/README.md

    - **Type:** string (enumerated values as per ST 2110-21, such as '2110TPNL' for narrow linear senders)
    - **Target:** SDP attribute 'a=fmtp:' format-specific parameter 'TP'
    - **Applicability:** AMWA IS-04
- **Name:** urn:x-nmos:cap:transport:ipmx_pass_through_delay

>   - **Description:** Minimum and maximum delay pass-through delay values in microseconds supported for IPMX receiver.
>   - **Specification:** per [VSF TR-10-1][TR-10-1]
>     - **Type:** JSON object with 'minimum' and 'maximum' number attributes.
>     - **Target:** Used to calculate setting of ST-2110-10:2021 Link Offet Delay via 'ext_link_offset_delay' in AMWA IS-05 Transport Parameter.

Reviewer: I think **Type** should be just `integer` as this is the type of the parameter being constrained, not the type of the parameter constraint itself. Compare e.g. `sample_depth` above.

I wonder about changing **Target** to **Note** because this parameter constraint doesn't truly have a target parameter which it is constraining.

```suggestion
  - **Description:** Identifies minimum and maximum pass-through delay in microseconds supported by an IPMX receiver.
  - **Specification:** per [VSF TR-10-1][TR-10-1]
    - **Type:** integer
    - **Note:** Used to calculate setting of ST 2110-10:2021 Link Offset Delay via 'ext_link_offset_delay' in AMWA IS-05 Transport Parameter.
```

Reviewer 2: I agree with using "Note" instead of "Target" because we need a way of differentiate true constraints and general information about a receiver. Ideally only true constraints should use the urn:x-nmos:cap: namespace but at least reserving "Target" for true constraints could be enough.

Reviewer: Thanks, @alabou. I wondered if we might come up with a way to express this as targeting a parameter of the NMOS Controller/system as a whole, but it's awkward because it's the Link Offset Delay that is really the target.
